packages/@react-spectrum/dropzone/docs/DropZone.mdx

```diff
@@ -0,0 +1,338 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/dropzone';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/dropzone/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+
+```jsx import
+import {DropZone} from '@react-spectrum/dropzone';
+import {Heading} from '@react-spectrum/text';
+import {Content} from '@react-spectrum/view';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Text} from 'react-aria-components';
+import {FileTrigger} from 'react-aria-components';
+import {Button} from '@react-spectrum/button';
+```
+
+---
+category: Drag and drop
+---
+
+# DropZone
+
+<PageDescription>{docs.exports.DropZone.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DropZone']}
+  since="3.0.0" />
```

Comment:
I _think_ adding this will mark that DropZone was added to the monopackage since `3.0.0` which isn't what we want. I think you can omit this and it will display the proper prerelease version and then will update to the monopackage version it is released with when it exits pre-release